source/blog/2017-10-16-fall-in-for-finagle.md

```diff
@@ -0,0 +1,144 @@
+---
+layout: post
+title: 🍂 Fall-ing for Finagle 🍂
+published: true 
+post_author:
+  display_name: Christopher Coco 
+  twitter: cacoco
+tags: Releases, Finagle, Finatra, Util, Scrooge, TwitterServer
+---
+
+As we move headlong into Autumn we thought this might be a good time to take
+a quick look back at some of the work from the previous quarter in [Finagle](https://twitter.github.io/finagle/), [Finatra](https://twitter.github.io/finatra/), [Scrooge](https://twitter.github.io/scrooge), [TwitterServer](https://twitter.github.io/twitter-server), and [Util](https://twitter.github.io/util) and preview what we're working on this quarter and beyond.
+
+This covers the
+[August](https://github.com/twitter/finagle/tree/finagle-7.0.0) and [September](https://github.com/twitter/finagle/tree/finagle-7.1.0) releases as well as the upcoming release planned for this month. As mentioned before, we plan to continue this review
+regularly on a quarterly basis. You can start with a recap of what we were talking about [last time](https://finagle.github.io/blog/2017/07/12/summer-review/).
+
+### News & Notes
+
+------------
+
+- Finagle's 7th "birthday" (7th anniversary of the [first public
+commits](https://github.com/twitter/finagle/tree/e04e51645374f8d958d85de384142dd00f4b7574))
+is approaching. Stay tuned for a celebratory blog post!
+
+- We're retiring the [Finatra](https://github.com/twitter/finatra)-specific
+blog, formerly available
+[here](https://twitter.github.io/finatra/blog/archives/). This will happen with 
+our next release scheduled in the next few weeks. Unfortunately, we won't be 
+keeping any of the old posts but any new information will be shared on the 
+finagle blog here.
+
+---------
+
+#### Efficiency
+
+Metrics ‘verbosity’ level. Similar to logging levels in a logging API, we've
+introduced the concept of a verbosity level for metrics exported by Finagle
+([1](https://github.com/twitter/finagle/commit/6c666ab5c3363ae5bd22b0fbd96f33995fe36ac7),
+[2](https://github.com/twitter/util/commit/5e03745015693bc61fa2aace8da5eb452ff6e53d),
+[3](https://github.com/twitter/util/commit/d08661fe49377aafd8af60231d6112d546e55e01),
+[4](https://github.com/twitter/util/commit/1b3ce1eddbb2a9af9fa4f97f1fc1b20ac3ee6a40))
+with the ability for users to configure their service's verbosity and to
+define a verbosity when creating a new metric.
+
+For more information see the [Util](https://twitter.github.io/util) User's Guide section on [`Leveraging
+Verbosity
+Levels`](https://twitter.github.io/util/guide/util-stats/user_guide.html#leveraging-verbosity-levels).
+
+Our internal roll-out of our new load balancer, [Deterministic
+Aperture](https://github.com/twitter/finagle/blob/finagle-6.45.0/finagle-core/src/main/scala/com/twitter/finagle/loadbalancer/aperture/DeterministicOrdering.scala) continues and we’re excited about bringing this to the wider community soon!
+
+### Resiliency
```

Comment:
what do you think of renaming this to New Features and then putting it up above Efficiency.

also, i think we should call out that its moved out of alpha/experimental and into the rollout phase. we could ask folks to try it if they are interested.

Reply:
It's hard to see here what the "this" is referring to move? The deterministic aperture section or all the of the resiliency section? Sorry.